src/pages/community/members.mdx

```diff
@@ -25,7 +24,7 @@ Contributions come in many forms, and we appreciate all of them! Examples includ
 
 Whatever your area of expertise and your skill level, there may be valuable contributions you can make. Are you a graphic designer? A technical writer? Do you know how to make videos? There might be a cool contribution in your future.
 
-We try and make contributions easy. For example, you can suggest documentation fixes by clicking on the Suggest Edits link on any page. And you can report bugs by opening a [GitHub issue](https://github.com/pantsbuild/pants/issues). If you want to hack on the Pants codebase itself there is a <Link to={`/${versions[0]}/docs/contributions`}>helpful guide</Link>.
+We try and make contributions easy. For example, you can suggest documentation fixes by clicking on the Suggest Edits link on any page. And you can report bugs by opening a [GitHub issue](https://github.com/pantsbuild/pants/issues). If you want to hack on the Pants codebase itself there is a <VersionedDocsLink version="current-dev" unversionedPath="docs/contributions">helpful guide</VersionedDocsLink>.
```

Comment:
As noted in the PR description this is changing from the first entry of `versions.json` to the "current". This means switching from the most recent `versioned_docs/...` entry (which corresponds to some `2.*.x` branch in https://github.com/pantsbuild/pants) to the contents of `docs/` (which corresponds to `main`).

I think this is reasonable: people are generally contributing to the `main` branch, so the docs there are what's relevant.

(For this particular one, arguably the "process" parts of the docs like https://www.pantsbuild.org/2.20/docs/contributions are "evergreen" pages and could be moved to this repo, while some pages like https://www.pantsbuild.org/2.20/docs/contributions/development/developing-rust are more strongly tied to a particular version and so aren't evergreen.)